packages/@okta/vuepress-site/docs/concepts/session/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Session management
+meta:
+  - name: description
+    content: An overview of session management in Okta. Learn about sessions in Okta and what kind of session logout to implement.
+---
+
+## Session management with Okta
+
+An Identity Provider (IdP) is a service that stores and manages digital identities. Okta is an IdP service that stores and verifies user identities, and is able to provide a user with a single set of sign-in credentials, allowing the user to access multiple applications. A session is a group of user interactions with a resource, such as a IdP service or an application (app), for a given amount of time. Using Okta as your IdP, there are two types of sessions that you need to consider for managing the user experience&mdash;the IdP session and the application session.
+
+### IdP session
+
+The IdP session is created by Okta after the user is authenticated using their credentials and various MFA options. Once authenticated, the user is able to access apps within the managed identity organization (org) using SSO, all within the scope of the IdP session. The session uses cookies to store user information to track and facilitate interactions between the user and the org. The IdP session is active until the user signs out of the org or when the session expires, based on the policies defined in the org.
+
+### Application session
+
+The application session (also known as a local session) is created by the app after the user is authenticated. This session uses cookies to store user app-specific information to track and facilitate interactions between the user and the app. The application session time frame depends on the nature of the app:
+* for privileged access apps, such as an e-commerce site where users make payments, a shorter and restrictive session is required
+* for non-privileged access apps, such as a catalog site where users browse listed items, the session can live longer and remain active even when the user navigates away, closes the browser tab or website, or closes the mobile app
```

Comment:
for both bullet items: 
1. for > For
2. Add a period at the end of the sentence.

Reply:
Susan's rewrite is better.